src/views/admin/AdminView.tsx

```diff
@@ -23,20 +24,63 @@ export const AdminView = () => {
       window.location.reload()
     }
   }
+
+  const handleExportClick = () => {
+    const fileName = 'atlas_local_state'
+    const state = Object.keys(localStorage).reduce((obj, k) => {
+      return { ...obj, [k]: localStorage.getItem(k) }
+    }, {})
+
+    const json = JSON.stringify(state)
+    const blob = new Blob([json], { type: 'application/json' })
+    const href = URL.createObjectURL(blob)
+    const link = document.createElement('a')
+    link.href = href
+    link.download = fileName + '.json'
+    document.body.appendChild(link)
+    link.click()
+    document.body.removeChild(link)
+  }
+
+  const handleFileChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (!e.target.files || !e.target.files.length) {
+      return
+    }
+    setFile(e.target.files[0])
+  }
+
+  const handleImportClick = async () => {
+    if (!file) {
+      return
+    }
+    const text = await file.text()
+    const state = JSON.parse(text)
+    Object.keys(state).forEach((key) => localStorage.setItem(key, state[key]))
+  }
+
   return (
-    <div>
-      <Text variant="h2">Choose environment</Text>
-      <Select
-        items={items}
-        onChange={handleChange}
-        value={value}
-        css={css`
-          margin-top: 20px;
-          max-width: 400px;
-        `}
-      />
-      <br />
+    <>
+      <div>
+        <Text variant="h2">Choose environment</Text>
+        <Select
+          items={items}
+          onChange={handleChange}
+          value={value}
+          css={css`
+            margin-top: 20px;
+            max-width: 400px;
+          `}
+        />
+      </div>
+      <div>
+        <Text variant="h2">Import/Export Local state</Text>
+        <Button onClick={handleExportClick}>Export local state</Button>
+        <div style={{ margin: '20px 0' }}>
+          <input type="file" onChange={handleFileChange} accept="application/json" />
```

Comment:
Could we make this input "virtual" or hidden so it's not visible on the screen but gets triggered when the user clicks import button? Should make the screen more readable